tests/functional/lang/parse-fail-undef-var-2.err.exp

```diff
@@ -1,5 +1,6 @@
 error: syntax error, unexpected ':', expecting '}'
```

Comment:
Again not a blocker for this PR, but it would be nice if this said `expecting ',', '?', or '}'`.